sources/packages/web/src/components/aest/students/modals/AddManualOverawardDeduction.vue

```diff
@@ -37,6 +37,14 @@
               ),
           ]"
         />
+        <v-textarea
+          v-model="formModel.overawardNotes"
+          variant="outlined"
+          label="Notes"
+          :rules="[(v) => checkNotesLengthRule(v, 'Notes')]"
+          required
+          class="mt-4 mb-n4"
```

Comment:
I am not sure why the `mb-n4` is needed here (I saw other places using it as well).
In general, we are trying to set also the `hide-details="auto"` to avoid unnecessary spaces below the text area prior to the validation.

Reply:
Yeah, I didn't that. The component should "hide details" by default IMO. Thanks. I'll remove the negative margin and and the property.

Reply:
Yes, agree 😉